app/build.gradle

```diff
@@ -33,9 +33,11 @@ android {
         resValue "bool", "IS_DEBUG", "false"
         buildConfigField "boolean", "USE_RELEASE_VERSIONING", "false"
         buildConfigField "String", "AMO_COLLECTION", "\"7e8d6dc651b54ab385fb8791bf9dac\""
+        def deepLinkSchemeValue = "fenix-dev"
```

Comment:
Oh nice! I didn't think this would work for some reason.

Reply:
Me too.  Luckily found this: https://developer.android.com/studio/build/gradle-tips#share-properties-with-the-manifest

Reply:
👏